doc/tutorials/13-charged_rod/13-charged_rod.ipynb

```diff
@@ -0,0 +1,270 @@
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\n",
+    "# Rod length\n",
+    "rod_length = 25.\n",
+    "\n",
+    "# line charge density of the rod \n",
+    "line_charge_dens =  2\n",
+    "\n",
+    "# rod radius\n",
+    "rod_radius = 1.0\n",
+    "\n",
+    "# Bjerrum length\n",
+    "bjerrum_length = 1.0\n",
+    "\n",
+    "# ion diameter\n",
+    "ion_diameter = 1.0\n",
+    "\n",
+    "# valency of the counterions\n",
+    "ion_valency = 1\n",
+    "\n",
+    "# number of beads of the rod\n",
+    "N_rod_beads = 50\n",
+    "\n",
+    "\n",
+    "# accuracy of p3m algorithm \n",
+    "p3m_accuracy =  1e-3;"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# particle types \n",
+    "rod_type = 1\n",
+    "ion_type = 2\n",
+    "\n",
+    "# total charge of the rod\n",
+    "total_rod_charge = line_charge_dens*rod_length\n",
+    "\n",
+    "# charge per rod bead\n",
+    "rod_charge_per_bead = -total_rod_charge/N_rod_beads\n",
+    "\n",
+    "# number of counterions\n",
+    "N_ions = int(total_rod_charge/ion_valency)\n",
+    "\n",
+    "#TODO assert neutrality\n",
+    "\n",
+    "\n",
+    "system = System(box_l=3*[rod_length])\n",
+    "\n",
+    "system.time_step = 0.01\n",
+    "system.cell_system.skin = 0.4\n",
+    "\n",
+    "system.thermostat.set_langevin(kT=1.0, gamma=0.5, seed=3)\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "wca_epsilon = 1.0\n",
+    "\n",
+    "#ion-ion interaction\n",
+    "system.non_bonded_inter[ion_type,ion_type].wca.set_params(\n",
+    "      epsilon=wca_epsilon, sigma=ion_diameter)\n",
+    "\n",
+    "# ion-rod interaction\n",
+    "system.non_bonded_inter[ion_type,rod_type].wca.set_params(\n",
+    "      epsilon=wca_epsilon, sigma=ion_diameter/2. + rod_radius)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "for idx in range(N_rod_beads):\n",
+    "    system.part.add(pos=[rod_length/2.,rod_length/2.,idx/N_rod_beads], type=rod_type, q=rod_charge_per_bead, fix=3*[True])\n",
+    "#TODO: comment on fix\n",
+    "\n",
+    "for _ in range(N_ions):\n",
+    "    system.part.add(pos=np.random.random(3) * system.box_l, type=ion_type, q=ion_valency)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "force_cap=1\n",
+    "system.force_cap = force_cap\n",
+    "\n",
+    "for i in range(4000):\n",
+    "    system.integrator.run(200)\n",
+    "    dist=system.analysis.min_dist(p1=[ion_type],p2=[ion_type, rod_type])\n",
+    "    print( f\"Warm up step {i}: min_dist={dist} force_cap={force_cap}\")\n",
+    "    force_cap = force_cap + 1.\n",
+    "    system.force_cap = force_cap\n",
```

Comment:
is this manual warmup necessary? tutorial 02 part 1 shows how to use the steepest descent algorithm to equilibrate the system

Reply:
We should go for best practice in all tutorials and have a steepest descent with energy/force convergence criterion, see #3881

Reply:
just realized tutorial 02 part 1 uses the `minimize_energy` module, which is deprecated...

Reply:
according to @schlaicha this tutorial 13 will replace tutorial 02 part 1